### PR TITLE
`timestampField` configuration field removal

### PR DESCRIPTION
**Pull request by nablaone**
Changes: 
- remove `timestampField` configuration parameter
-  `@timestamp` field is resolved in a query transformation pipeline now

### CODE REVIEW

File: quesma/model/typical_queries/hits.go [hunk 1]
```diff
@@ -29,6 +29,7 @@ type Hits struct {
 	addScore       bool // true <=> we add hit.Score field to the response (whose value is always 1)
 	addVersion     bool // true <=> we add hit.Version field to the response (whose value is always 1)
 	indexName      string
+	useDefaultID   bool
```
Comment (mieciu) [inline quesma/model/typical_queries/hits.go:32]: I think it now makes sense to place a little comment explaining what the default ID is in this context

File: quesma/clickhouse/table_discovery.go [hunk 2]
```diff
@@ -189,14 +190,11 @@ func (td *tableDiscovery) autoConfigureTables(tables map[string]map[string]strin
...
-		} else {
-			configuredTables[table] = discoveredTable{table, columns, config.IndexConfiguration{}, comment, createTableQuery}
-		}
+		configuredTables[table] = discoveredTable{table, columns, config.IndexConfiguration{}, comment, createTableQuery, maybeTimestampField}
+
 	}
-	for tableName, conf := range configuredTables {
-		autoDiscoResults.WriteString(fmt.Sprintf("{table: %s, timestampField: %s}, ", tableName, conf.config.GetTimestampField()))
+	for tableName := range configuredTables {
+		autoDiscoResults.WriteString(fmt.Sprintf("{table: %s}, ", tableName))
```
Comment (mieciu) [inline quesma/clickhouse/table_discovery.go:197]: As this is auto discovery mode I think it's still worth printing TS field.

File: quesma/model/typical_queries/hits.go [hunk 3]
```diff
@@ -132,12 +133,19 @@ func (query Hits) addAndHighlightHit(hit *model.SearchHit, resultRow *model.Quer
...
+
 func (query Hits) computeIdForDocument(doc model.SearchHit, defaultID string) string {
-	tsFieldName, err := query.table.GetTimestampFieldName()
-	if err != nil {
+
+	if query.useDefaultID {
 		return defaultID
 	}
 
+	tsFieldName := model.TimestampFieldName
```
Comment (mieciu) [inline quesma/model/typical_queries/hits.go:147]: As we've chatted on Zoom, this isn't correct as it always takes `@timestamp`

File: quesma/model/typical_queries/hits.go (hunk 1 above)
Comment (nablaone) [reply quesma/model/typical_queries/hits.go]: It relies on the presence of a timestamp field now.

File: quesma/model/typical_queries/hits.go (hunk 3 above)
Comment (nablaone) [reply quesma/model/typical_queries/hits.go]: It is fixed now.

**Review by mieciu — APPROVED** [review]
LGTM now!

**Pull request merged**